apps/system-e2e/src/tests/judicial-system/smoke/custody-prosecutor.spec.ts

```diff
@@ -43,7 +43,7 @@ test.describe('Custody Prosecutor', () => {
     await page
       .locator('input[name=defenderEmail]')
       .fill('jl-auto-defender@kolibri.is')
-    await page.getByRole('checkbox').last().check()
+    await page.locator('input[name=defender-access-no]').click()
```

Comment:
```suggestion
    await page.getByRole('input', { name: 'defender-access-no' }).click()
```

Reply:
I just tried with getByRole and I can't get it to work :(